apps/reference/docs/guides/with-nuxt-3.mdx

```diff
@@ -8,6 +8,12 @@ sidebar_label: Nuxt 3
 import Tabs from '@theme/Tabs'
 import TabItem from '@theme/TabItem'
 
+:::caution
+
+This example uses Nuxt Supabase v0, that depends on outdated supabase-js v1. It will be updated when updated Nuxt Supabase is released.
```

Comment:
@silentworks just checking that you meant v0 here

Reply:
Yeah this is from Nuxt Supabase which started from version 0.x which under the hood makes use of supabase-js v1.